Add Delivery 2 "Report" section (#3701)
SumoLogic/sumologic-documentation: docs/cloud-soar/main-menu.md

      1. Select **Dashboard** in the upper-left corner of the UI. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-dashboards.png')} alt="Access dashboards" width="300"/>
      1. Select a dashboard.
      1. Click the **Edit** button. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-edit-dashboard-button.png')} alt="Empty dashboard" width="150"/><br/>
      :::note
      Widgets are shared between **Reports** and **Dashboards**.

>       To edit a widget user can click the gear icon in the upper-right corner, then select **Report** or through **Dashboard** edit mode.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/> 

Author: ```suggestion
      To edit a widget, you can click the gear icon in the upper-right corner and select **Report**. You can also use the **Dashboard** edit mode.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/> 
```

>       To edit a widget user can click the gear icon in the upper-right corner, then select **Report** or throught **Dashboard** edit mode.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/> 

Author: ```suggestion
      To edit a widget user can click the gear icon in the upper-right corner, then select **Report** or through **Dashboard** edit mode.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/> 
```

      1. Select **Dashboard** in the upper-left corner of the UI. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-dashboards.png')} alt="Access dashboards" width="300"/>
      1. Select a dashboard.
      1. Click the **Edit** button. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-edit-dashboard-button.png')} alt="Empty dashboard" width="150"/><br/>
      :::note

>       To open widgets from Reports, click the gear icon in the upper-right corner of the UI, then select **Report**. <br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/> 

Reviewer: ```suggestion
      Widgets are shared between **Reports** and **Dashboards**.
      To edit a widget user can click the gear icon in the upper-right corner, then select **Report** or throught **Dashboard** edit mode.<br/><img src={useBaseUrl('img/cloud-soar/delivery-2-access-reports.png')} alt="Access reports" width="150"/> 
```
